src/backend/base/langflow/components/nvidia/nemo_evaluator.py

```diff
@@ -191,26 +167,121 @@ class NVIDIANeMoEvaluatorComponent(Component):
             min_label="Precise",
             max_label="Creative",
             value=0.1,
-            info="The temperature to be used during generation sampling (0.0 to 2.0).",
+            info="The temperature to be used during generation sampling (0.0 to 2.0)."
         ),
-        IntInput(name="313_top_k", display_name="Top_k", info="Top_k value for generation sampling.", value=1),
-        # Conditional input for run_inference = False
-        StrInput(
-            name="320_output_file",
-            display_name="Output File",
-            info="Path of the output file in NeMo Data Store in nds:<dataset>/<json output file>.",
+        IntInput(
+            name="313_top_k",
+            display_name="Top_k",
+            info="Top_k value for generation sampling.",
+            value=1
+        ),
+        MessageTextInput(
+            name="dataset",
+            display_name="Dataset",
+            info="Enter the dataset ID or name used to train the model",
+            value="testing",
+        ), DataInput(
+            name="evaluation_data",
+            display_name="Training Data",
+            is_list=True,
         ),
     ]
 
-    def fetch_models(self):
-        """Fetch models from the specified API endpoint and return a list of model names."""
-        model_url = f"{self.model_base_url}/v1/models"
+    # Inputs for Custom Evaluation
+    llm_as_judge = [
```

Comment:
I believe we are going to want to reuse the numbering system as with the options for the other evaluation types, with prefixes for these starting with 400_. Unless the Langflow folks have some recommendation on a better way, we need this for the options to show up in the correct order when we are dynamically adding/removing options.

Reply:
If you try switching back and forth between different evaluation types, what happens?